ext/pdo/pdo.c

```diff
@@ -129,10 +132,22 @@ PDO_API void php_pdo_unregister_driver(const pdo_driver_t *driver) /* {{{ */
 		return;
 	}
 
+	zend_hash_str_del(&pdo_driver_specific_ce_hash, driver->driver_name, driver->driver_name_len);
 	zend_hash_str_del(&pdo_driver_hash, driver->driver_name, driver->driver_name_len);
 }
 /* }}} */
 
+PDO_API zend_result php_pdo_register_driver_specific_ce(const pdo_driver_t *driver, zend_class_entry *ce) /* {{{ */
+{
+	if (!zend_hash_str_exists(&module_registry, "pdo", sizeof("pdo") - 1)) {
+		zend_error_noreturn(E_ERROR, "You MUST load PDO before loading any PDO drivers");
```

Comment:
This error message should maybe be along the lines:
```suggestion
		zend_error_noreturn(E_ERROR, "The PDO extension must be loaded first in order to load PDO drivers");
```
?

Reply:
Yeah, makes a lot of sense to me :)